modules/installation-user-infra-exporting-common-variables.adoc

```diff
@@ -54,6 +67,20 @@ $ export REGION=`jq -r .gcp.region <installation_directory>/metadata.json`
 ----
 <1> For `<installation_directory>`, specify the path to the directory that you stored the installation files in.
 
+ifdef::shared-vpc[]
+. Export the variable for the name of the account that hosts the shared VPC:
```

Comment:
It should be the **email** of the service account that belongs to host project

Reply:
It's weird that this variable doesn't include `_EMAIL` like the other variables that expect an email address.